server/src/main/java/com/linecorp/centraldogma/server/internal/api/ProjectServiceV1.java

```diff
@@ -93,134 +94,62 @@ public ProjectServiceV1(ProjectManager projectManager, CommandExecutor executor,
-                  .thenCompose(projectInfo -> execute(Command.createProject(author, projectInfo.name())))
-                  .handle((unused, thrown) -> {
-                      try {
-                          if (thrown == null) {
-                              return DtoConverter.convert(projectManager().get(request.name()));
-                          } else {
-                              return Exceptions.throwUnsafely(thrown);
-                          }
-                      } finally {
-                          if (thrown != null) {
-                              // Remove created project from metadata.
-                              mds.removeProject(request.name(), author);
-                          }
-                      }
-                  });
+        return execute(Command.createProject(author, request.name()))
+                .handle(returnOrThrow(() -> DtoConverter.convert(projectManager().get(request.name()))));
     }
 
     /**
-     * DELETE /projects/{projectName}
+     * GET /projects/{projectName}
      *
-     * <p>Removes a project.
+     * <p>Gets the {@link ProjectMetadata} of the specified {@code projectName}.
      */
-    @Delete("/projects/{projectName}")
-    public CompletableFuture<Void> removeProject(@RequestObject Project project,
-                                                 @RequestObject Author author) {
-        return execute(Command.removeProject(author, project.name()))
-                .thenCompose(unused -> mds.removeProject(project.name(), author))
-                .handle(HttpApiUtil::throwUnsafelyIfNonNull);
-    }
-
-    /**
-     * PATCH /projects/{projectName}
-     *
-     * <p>Patches a project with the JSON_PATCH. Currently, only unremove project operation is supported.
-     */
-    @ConsumeType("application/json-patch+json")
-    @Patch("/projects/{projectName}")
-    public CompletionStage<ProjectDto> patchProject(@Param("projectName") String projectName,
-                                                    @RequestObject JsonNode node,
-                                                    @RequestObject Author author) {
-        checkUnremoveArgument(node);
-        return mds.unremoveProject(projectName, author)
-                  .thenCompose(projectInfo -> execute(Command.unremoveProject(author, projectName)))
-                  .handle(returnOrThrow(() -> DtoConverter.convert(projectManager().get(projectName))));
+    @Get("/projects/{projectName}")
+    @Decorator(ProjectMembersOnly.class)
+    public CompletionStage<ProjectMetadata> getProjectMetadata(@Param("projectName") String projectName) {
+        return mds.getProject(projectName);
     }
 
-    // TODO(hyangtack) The following APIs would return ProjectInfo to the client. It is different action
-    //                 from the above API which is returning ProjectDto. We would handle this later.
-
     /**
-     * POST /projects/{projectName}/members/{user}
+     * GET /projects/{projectName}/accessibility
```

Comment:
The term 'accessibility' is often used for disabled people. Do we have a better term here?

Reply:
I will remove this API, then add `checkPermissionOnly=true` parameter to `GET /projects/{projectName}` API.